pkg/proto/datadog/process/workloadmeta_process.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package datadog.process;
+
+option go_package = "pkg/proto/pbgo"; // golang
+
+message WorkloadMetaProcessEvents {
+  int32 eventID = 1;
+  repeated ProcessEventSet setEvents = 2;
+  repeated ProcessEventUnset unsetEvents = 3;
```

Comment:
I think that during our brainstorm meeting we wanted to use `oneof`

```
message ProcessEvent {
    oneof operation {
      ProcessEventSet set = 2;
      ProcessEventUnset unset = 3;
    }
}

message WorkloadMetaProcessEvents {
  int32 eventID = 1;
  repeated ProcessEvent = 2;
}

```

Reply:
Is there a reason why we split Set and Unset events ? Perhaps the `pid` could be mandatory and the other fields could be optional ?

Reply:
Hi! Sorry for the late reply! @mbotarro and I were talking about this and decided this format would simply be easier to parse than using `oneOf`. I spoke to @clamoriniere offline and we agreed that this would be fine.